x-pack/plugins/cases/common/api/cases/configure.ts

```diff
@@ -34,24 +34,36 @@ const CasesConfigureBasicRt = rt.intersection([
   }),
 ]);
 
-export const CasesConfigureRequestRt = CasesConfigureBasicRt;
-export const CasesConfigurePatchRt = rt.intersection([
-  rt.partial(CasesConfigureBasicWithoutOwnerRt.props),
+export const ConfigurationRequestRt = CasesConfigureBasicRt;
+export const ConfigurationPatchRequestRt = rt.intersection([
+  rt.partial(ConfigurationBasicWithoutOwnerRt.props),
   rt.type({ version: rt.string }),
 ]);
 
-export const CaseConfigureAttributesRt = rt.intersection([
+const ConfigurationActivityFieldsRt = rt.type({
+  created_at: rt.string,
+  created_by: UserRt,
+  updated_at: rt.union([rt.string, rt.null]),
+  updated_by: rt.union([UserRt, rt.null]),
+});
+
+export const ConfigurationPartialAttributesRt = rt.intersection([
```

Comment:
@cnasikas unfortunately I wasn't able to get an equivalent of the function you had for converting a type into partial working. It seemed like it was because this type was doing multiple intersections. When I tried to do `type.props` props doesn't exist on the result.

Reply:
Also would we rather this type be within server instead of common since that's the only place it is used? If so I'll need to export a few of these types that's all.

Reply:
I agree, let's move it within the server folder.